save-agent/save-agent-common-api/build.gradle.kts

```diff
@@ -0,0 +1,79 @@
+import com.saveourtool.save.buildutils.configureSigning
+
+@Suppress("DSL_SCOPE_VIOLATION", "RUN_IN_SCRIPT")  // https://github.com/gradle/gradle/issues/22797
+plugins {
+    kotlin("multiplatform")
+    alias(libs.plugins.kotlin.plugin.serialization)
+    id("com.saveourtool.save.buildutils.code-quality-convention")
+    id("com.saveourtool.save.buildutils.publishing-configuration")
+}
+kotlin {
+    jvm {
+        compilations.all {
+            kotlinOptions {
+                jvmTarget = Versions.jdk
+            }
+        }
+    }
+    jvmToolchain {
+        this.languageVersion.set(JavaLanguageVersion.of(Versions.jdk))
+    }
+
+    js(IR) {
```

Comment:
we need `js` and macosX64 because we use some classes from API in save-cloud-common